lib/td/client/api.rb

```diff
@@ -121,7 +121,12 @@ def self.normalized_msgpack(record, out = nil)
         record[k] = v.to_s
       end
     }
-    record.to_msgpack(out)
+    if out
+      out << record.to_msgpack
+      out
+    else
+      record.to_msgpack
+    end
```

Comment:
Just curious: doesn't it work with msgpack on JRuby? Generally the previous code is better (less code branch.) Though I don't care the change because td doesn't use this method with 'out'.